"use strict";(self.webpackChunkegc_analysis_docs=self.webpackChunkegc_analysis_docs||[]).push([[976],{2053:(e,t,o)=>{o.r(t),o.d(t,{assets:()=>d,contentTitle:()=>r,default:()=>c,frontMatter:()=>i,metadata:()=>s,toc:()=>l});const s=JSON.parse('{"id":"intro","title":"Getting Started","description":"Who are you?","source":"@site/docs/intro.md","sourceDirName":".","slug":"/intro","permalink":"/docs/intro","draft":false,"unlisted":false,"editUrl":"https://github.com/EpigenomicsCode/egc-analysis-docs/docs/intro.md","tags":[],"version":"current","lastUpdatedBy":"Olivia Wen-Mei Lang","lastUpdatedAt":1759817739000,"sidebarPosition":1,"frontMatter":{"sidebar_position":1},"sidebar":"tutorialSidebar","next":{"title":"ChIP-exo","permalink":"/docs/ChIP-exo/"}}');var a=o(4848),n=o(8453);const i={sidebar_position:1},r="Getting Started",d={},l=[{value:"Who are you?",id:"who-are-you",level:2},{value:"I have <code>.bam</code> files",id:"i-have-bam-files",level:4},{value:"I have sequencing data, but no <code>.bam</code> files",id:"i-have-sequencing-data-but-no-bam-files",level:4},{value:"I do not have sequencing data",id:"i-do-not-have-sequencing-data",level:4},{value:"\ud83e\udd37\u200d\u2640\ufe0f Now what?",id:"\ufe0f-now-what",level:2},{value:"Layout and tutorial organization",id:"layout-and-tutorial-organization",level:2},{value:"Browse tutorials (by genomic assay)",id:"browse-tutorials-by-genomic-assay",level:3},{value:"Discussions",id:"discussions",level:3}];function h(e){const t={a:"a",code:"code",h1:"h1",h2:"h2",h3:"h3",h4:"h4",header:"header",p:"p",...(0,n.R)(),...e.components};return(0,a.jsxs)(a.Fragment,{children:[(0,a.jsx)(t.header,{children:(0,a.jsx)(t.h1,{id:"getting-started",children:"Getting Started"})}),"\n",(0,a.jsx)(t.h2,{id:"who-are-you",children:"Who are you?"}),"\n",(0,a.jsxs)(t.h4,{id:"i-have-bam-files",children:["I have ",(0,a.jsx)(t.code,{children:".bam"})," files"]}),"\n",(0,a.jsxs)(t.p,{children:["You've gotten some sequencing data off the sequencer, done some QC filtering and maybe even trimmed some of your reads, and now you have performed your aligments and are left with a bunch of ",(0,a.jsx)(t.code,{children:".bam"})," files."]}),"\n",(0,a.jsxs)(t.h4,{id:"i-have-sequencing-data-but-no-bam-files",children:["I have sequencing data, but no ",(0,a.jsx)(t.code,{children:".bam"})," files"]}),"\n",(0,a.jsx)(t.p,{children:"If you haven't done this part yet, there are tons of tutorials out there that can help you get to this stage so Google up some resources to help get you to this part. There are better resources out there to help you with this so we won't try to cover it in these docs. We encourage you to understand the assumptions and parameters of your pipeline to this point because there are certain scenarios where this may influence your parameter choices in using ScriptManager."}),"\n",(0,a.jsx)(t.h4,{id:"i-do-not-have-sequencing-data",children:"I do not have sequencing data"}),"\n",(0,a.jsxs)(t.p,{children:["If you don't have data yet but wish to explore genomic data analysis in these tutorials anyway, consider downloading data from the ",(0,a.jsx)(t.a,{href:"https://www.encodeproject.org/",children:"ENCODE portal"})," to build your own questions and do your own hypothesis testing exploration."]}),"\n",(0,a.jsx)(t.h2,{id:"\ufe0f-now-what",children:"\ud83e\udd37\u200d\u2640\ufe0f Now what?"}),"\n",(0,a.jsx)(t.p,{children:"This is the fun part but also the most challenging part: creatively considering what kinds of comparisons to make that will answer the questions you are trying to ask of your data and that will confirm or reject your hypotheses. The best place to start is to mimic existing analysis strategies so we have compiled a set of examples in this collection of tutorials. Feel free to mix-and-match parts of these workflows to customize your analysis."}),"\n",(0,a.jsx)(t.h2,{id:"layout-and-tutorial-organization",children:"Layout and tutorial organization"}),"\n",(0,a.jsx)(t.h3,{id:"browse-tutorials-by-genomic-assay",children:"Browse tutorials (by genomic assay)"}),"\n",(0,a.jsx)(t.p,{children:"We have roughly grouped our tutorials by type of genomics assay for users coming to this resource with specific datasets in mind. Keep in mind that some of these tutorials can work for different types of genomic assays and so the same or similar tutorials may be listed for several different assays. These tutorials are by no means comprehensive and just because an analysis isn't explicitly grouped in with your assay, it doesn't mean you can't use that analysis for your own work. The pipeline building possibilities are endless!"}),"\n",(0,a.jsx)(t.h3,{id:"discussions",children:"Discussions"}),"\n",(0,a.jsx)(t.p,{children:"We will also include some more general discussions around normalization and replicate handling that are frequently asked to help users to make a choice that best works for their data and their questions. These tend to be more open-ended guidelines that don't work for everyone's scenarios but we have found prompts useful discussion for helping users to understand the important considerations for their analysis."})]})}function c(e={}){const{wrapper:t}={...(0,n.R)(),...e.components};return t?(0,a.jsx)(t,{...e,children:(0,a.jsx)(h,{...e})}):h(e)}},8453:(e,t,o)=>{o.d(t,{R:()=>i,x:()=>r});var s=o(6540);const a={},n=s.createContext(a);function i(e){const t=s.useContext(n);return s.useMemo(function(){return"function"==typeof e?e(t):{...t,...e}},[t,e])}function r(e){let t;return t=e.disableParentContext?"function"==typeof e.components?e.components(a):e.components||a:i(e.components),s.createElement(n.Provider,{value:t},e.children)}}}]);