"use strict";(self.webpackChunkegc_analysis_docs=self.webpackChunkegc_analysis_docs||[]).push([[936],{5342:(e,o,s)=>{s.r(o),s.d(o,{assets:()=>l,contentTitle:()=>h,default:()=>u,frontMatter:()=>r,metadata:()=>t,toc:()=>c});const t=JSON.parse('{"id":"ChIP-exo/index","title":"ChIP-exo Tutorials","description":"Also \\"ChIP-nexus\\"","source":"@site/docs/ChIP-exo/index.md","sourceDirName":"ChIP-exo","slug":"/ChIP-exo/","permalink":"/docs/ChIP-exo/","draft":false,"unlisted":false,"editUrl":"https://github.com/EpigenomicsCode/egc-analysis-docs/docs/ChIP-exo/index.md","tags":[],"version":"current","lastUpdatedBy":"Olivia Wen-Mei Lang","lastUpdatedAt":1759818079000,"sidebarPosition":3,"frontMatter":{"sidebar_label":"ChIP-exo","sidebar_position":3},"sidebar":"tutorialSidebar","previous":{"title":"Getting Started","permalink":"/docs/intro"},"next":{"title":"Peak calling (ChExMix)","permalink":"/docs/ChIP-exo/peak-calling-chexmix"}}');var i=s(4848),n=s(8453),a=s(8774);const r={sidebar_label:"ChIP-exo",sidebar_position:3},h="ChIP-exo Tutorials",l={},c=[{value:"Read more:",id:"read-more",level:2},{value:"Which tutorial should I start with?",id:"which-tutorial-should-i-start-with",level:2},{value:"Which RefPTs should I use?",id:"which-refpts-should-i-use",level:2},{value:"How should I handle replicates?",id:"how-should-i-handle-replicates",level:2}];function d(e){const o={a:"a",em:"em",h1:"h1",h2:"h2",header:"header",p:"p",strong:"strong",...(0,n.R)(),...e.components};return(0,i.jsxs)(i.Fragment,{children:[(0,i.jsx)(o.header,{children:(0,i.jsx)(o.h1,{id:"chip-exo-tutorials",children:"ChIP-exo Tutorials"})}),"\n",(0,i.jsx)(o.p,{children:(0,i.jsx)(o.em,{children:'Also "ChIP-nexus"'})}),"\n",(0,i.jsx)("div",{class:"tutorial-img-flow-container",children:(0,i.jsx)("img",{src:"/img/assays/ChIP-exo.svg",alt:"ChIP-exo cartoon",height:"300"})}),"\n",(0,i.jsx)(o.h2,{id:"read-more",children:"Read more:"}),"\n",(0,i.jsxs)(a.A,{href:"https://enseqlopedia.com/wiki-entry/dna-sequencing-methods/dna-protein-interactions/chip-seqht-chipchip-exomint-chip/",className:"button button--primary",children:[" ChIP-Seq/HT-ChIP/ChIP-exo/Mint-ChIP ",(0,i.jsx)(o.em,{children:"(Enseqlopedia)"})," "]}),"\n",(0,i.jsxs)(a.A,{href:"https://pubmed.ncbi.nlm.nih.gov/23026909/",className:"button button--primary",children:[" ChIP-exo 1.0 ",(0,i.jsx)(o.em,{children:"(Rhee 2012)"})," "]}),"\n",(0,i.jsxs)(a.A,{href:"https://pubmed.ncbi.nlm.nih.gov/30030442/",className:"button button--primary",children:[" ChIP-exo 5.0 ",(0,i.jsx)(o.em,{children:"(Rossi 2018)"})," "]}),"\n",(0,i.jsx)(o.h2,{id:"which-tutorial-should-i-start-with",children:"Which tutorial should I start with?"}),"\n",(0,i.jsxs)(o.p,{children:["We like for users unfamiliar with ScriptManager to start with the ",(0,i.jsx)(o.a,{href:"pileup",children:"heatmaps and composite plots tutorial"})," to get a quick feel for the types of figures you can make with your data. If you have not called peaks, eventually you may need to in order to have ",(0,i.jsx)(o.a,{href:"/docs/glossary",children:(0,i.jsx)(o.strong,{children:"RefPTs"})})," for your publication Tag"]}),"\n",(0,i.jsx)(o.h2,{id:"which-refpts-should-i-use",children:"Which RefPTs should I use?"}),"\n",(0,i.jsx)(o.p,{children:"It is common practice to use peak calls for general heatmap generation but with ChIP-exo being base-pair resolution data and peak callers having imperfect resolution, a more exact and high resolution reference point can give you more interesting insights into your data."}),"\n",(0,i.jsxs)(o.p,{children:["For ",(0,i.jsx)(o.a,{href:"/docs/glossary",children:(0,i.jsx)(o.strong,{children:"ssTFs"})}),", there are usually ",(0,i.jsx)(o.a,{href:"/docs/glossary",children:"weblogos"})," representing the position-specific nucleotide preferences of the ",(0,i.jsx)(o.a,{href:"/docs/glossary",children:"TFBS"}),". If you are ChIP-ing one of these targets, you have the opportunity to get some higher resolution RefPTs by obtaining every instance of some motif in the genome (use ",(0,i.jsx)(o.a,{href:"https://meme-suite.org/meme/meme_5.5.5/tools/fimo",children:(0,i.jsx)(o.strong,{children:"FIMO"})}),') to use as a RefPT for your heatmap and composite plots. You can even intersect your motif instances to keep the ones that overlap for a peak call to get the set of "bound" sites (obligatory disclaimer on thresholding caveats here). For ChIP-exo data, this will give you a better chance at observing the actual bp-resolution crosslinking profiles of a target.']}),"\n",(0,i.jsx)(o.h2,{id:"how-should-i-handle-replicates",children:"How should I handle replicates?"}),"\n",(0,i.jsxs)(o.p,{children:["This is an important question as there are many options available to you but some may create some ",(0,i.jsx)(o.em,{children:"circular reasoning"})," when you test your hypothesis. One thing we like to do to avoid this is to call peaks or do whatever manipulations you want to build your ",(0,i.jsx)(o.a,{href:"/docs/glossary",children:(0,i.jsx)(o.strong,{children:"RefPTs"})})," all based on one replicate. Then when you pileup and observe signal enrichment of the data, you can plot a different replicate that was held out from any ",(0,i.jsx)(o.a,{href:"/docs/glossary",children:(0,i.jsx)(o.strong,{children:"RefPT"})}),"-building to fairly perform comparisons."]}),"\n",(0,i.jsxs)(o.p,{children:["In other instances, you may need as much signal as you can get and you could consider ",(0,i.jsx)(o.a,{href:"https://pughlab.mbg.cornell.edu/scriptmanager-docs/docs/Tools/bam-manipulation/merge-bam",children:"merging your BAM files"}),"."]})]})}function u(e={}){const{wrapper:o}={...(0,n.R)(),...e.components};return o?(0,i.jsx)(o,{...e,children:(0,i.jsx)(d,{...e})}):d(e)}},8453:(e,o,s)=>{s.d(o,{R:()=>a,x:()=>r});var t=s(6540);const i={},n=t.createContext(i);function a(e){const o=t.useContext(n);return t.useMemo(function(){return"function"==typeof e?e(o):{...o,...e}},[o,e])}function r(e){let o;return o=e.disableParentContext?"function"==typeof e.components?e.components(i):e.components||i:a(e.components),t.createElement(n.Provider,{value:o},e.children)}}}]);